tests/dCommonTests/dEnumsTests/MagicEnumTests.cpp

```diff
@@ -6,55 +6,58 @@
 #include "eGameMessageType.h"
 #include "eWorldMessageType.h"
 #include <chrono>
+#include <string>
+
+#define log_test(e, y, z)\
+	LOG("%s %s", StringifiedEnum::ToString(static_cast<e>(y)).data(), #z);\
+	ASSERT_STREQ(StringifiedEnum::ToString(static_cast<e>(y)).data(), #z);
```

Comment:
You dont need a .data call here I dont believe?  You can just compare the stringview directly to a const char*?